.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
   "trailingComma": "es5",
   "tabWidth": 2,
-  "arrowParens": "always",
+  "arrowParens": "avoid",
   "singleQuote": true,
   "quoteProps": "as-needed",
   "bracketSpacing": true,
-  "printWidth": 120
+  "printWidth": 90
```

Comment:
Why 90?

Reply:
90 is a kinda safe number for 14" Laptop viewports and above. Tried 120 on my laptop but had to scroll right :c